vector/src/main/java/im/vector/app/features/roomprofile/notifications/RoomNotificationSettingsViewState.kt

```diff
@@ -40,12 +40,12 @@ data class RoomNotificationSettingsViewState(
  */
 val RoomNotificationSettingsViewState.notificationStateMapped: Async<RoomNotificationState>
     get() {
-        if ((roomSummary()?.isEncrypted == true && notificationState() == RoomNotificationState.MENTIONS_ONLY) ||
-                notificationState() == RoomNotificationState.ALL_MESSAGES) {
```

Comment:
Previously for `ALL_MESSAGES` it was always returning `ALL_MESSAGES_NOISY`, now it will be `ALL_MESSAGES` and this change is not mentioned in the issue. Is it intentional?

Reply:
no it's not intentional. 
I will revert this specification for ALL_MESSAGE.